src/util/GeoSparqlHelpers.h

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021, University of Freiburg,
+// Author: Hannah Bast <bast@cs.uni-freiburg.de>
+// Chair of Algorithms and Data Structures
+
+#ifndef QLEVER_GEOSPARQLHELPERS_H
+#define QLEVER_GEOSPARQLHELPERS_H
+
+#include <string>
+
+// TODO: How can I declare the lambdas in the .h file and implement them in the
+// .cpp file? The best I managed so far is define them with inline in the .f
+// file. If I define them without inline in the .cpp file, I get a multiple
+// definition error.
+//
+// namespace ad_utility {
+//
+// // Longitude of a WKT point (it's the first coordinate).
+// std::function<double(const std::string& p)> wktLongitude;
+//
+// // Latitude of a WKT point (it's the second coordinate).
+// std::function<double(const std::string& p)> wktLatitude;
+//
+// // Distance between two WKT points.
+// std::function<double(const std::string& p1, const std::string& p2)> wktDist;
+//
+// }  // namespace ad_utility
+
+// Helper function that checks whether a string looks like a WKT pointC (just
+// checks the beginning and that it has the minimal length).
+namespace {
+bool is_wkt_point(const std::string& p) {
+  return p.size() >= 12 && p[0] == '"' && (p[1] == 'P' || p[1] == 'p') &&
+         (p[2] == 'O' || p[2] == 'o') && (p[3] == 'I' || p[3] == 'i') &&
+         (p[4] == 'N' || p[4] == 'n') && (p[5] == 'T' || p[5] == 't') &&
+         p[6] == '(';
```

Comment:
In the header you shouldn't use the anonymous namespace, (even if it works) but the `inline` keyword (otherwise, the code for the function will probably be duplicated).
But if you want, you can put these two functions in the `.cpp` file.

Reply:
They are now in the .cpp file